src/widgets/sort-by/__tests__/sort-by-test.ts

```diff
@@ -106,11 +133,4 @@ describe('sortBy()', () => {
       },
     ]);
   });
-
-  it('sets the underlying index', () => {
-    widget.setIndex('index-b');
-
-    expect(helper.setIndex).toHaveBeenCalledTimes(1);
-    expect(helper.search).toHaveBeenCalledTimes(1);
-  });
```

Comment:
Should this be possible? As we don't expose `setIndex`, I feel like this test isn't needed

Reply:
it should be calling `getWidgetRenderState().refine` for an equivalent test

Reply:
however it makes sense that this is tested in the connector alone